backend/DAO/studentDAO.js

```diff
@@ -58,15 +58,22 @@ module.exports = {
             resolve(studentToSkills);
         });
     },
-    // getStudentsBySkills: (skillsId) => {
-    //     return database('student')
-    //         .select(['student.id', 'student.name'])
-    //         .leftJoin('student_has_skills AS shs', 'shs.student_id', 'student.id')
-    //         .whereIn('shs.skill_id', skillsId)
-    //         .groupBy('student.id')
-    //              
+
+    getStudentsBySkills: async (skillsId) => {
+        let skillsNumber = skillsId.length;
+        let result = await database('student_has_skills')
+            .select('student_id')
+            .whereIn('skill_id', skillsId)
+            
+            .groupBy('student_id')
+                 
                 //THIS SHOULD BE REPLACED WITH THE 50% CONDITION
-    //         .having(database.raw('count(shs.skill_id) > 2'))
-    // }
```

Comment:
lets remove comments like this one if they are not useful anymore

Reply:
I eliminated this line